packages/api-v4/src/aglb/types.ts

```diff
@@ -17,6 +17,13 @@ export interface CreateLoadbalancerPayload {
   configurations?: ConfigurationPayload[];
 }
 
+/**
+ * @TODO AGLB remove when we move to full creation flow
```

Comment:
I think we might be introducing different TODO formatting that will make this less easy to find via search. 👀 
This is more in line with the formatting we've used throughout the files. Minor, but staying consistent is useful.

```suggestion
 * TODO: AGLB - remove when we move to full creation flow
```

Reply:
On that note: we should capture the clean up in a ticket if we don't have one already, so we don't unintentionally leave this stuff around longer than we need.